build.gradle

```diff
@@ -17,44 +17,70 @@ plugins {
     id 'groovy'
     id 'idea'
     id 'java-gradle-plugin'
-    id "nebula.facet" version "3.4.0"
-    id 'com.gradle.plugin-publish' version '0.10.0'
+    id 'com.gradle.plugin-publish' version '0.20.0'
+}
+
+repositories {
+    mavenCentral()
 }
 
 group = 'org.gradle.crypto'
-version = '1.2.0'
+version = '2.0.0-SNAPSHOT'
 
-targetCompatibility = '1.7'
-sourceCompatibility = '1.7'
+targetCompatibility = JavaVersion.VERSION_11
+sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
You should stick with Java 8 given that this is the minimum Java version to run Gradle.